#An array of position and velocities of moving obstacles
Header header
int32 number_of_obstacles
leg_detector3/pos_vel[20] pos_vel
